firefox-ios/Client/Frontend/Library/Reader/ReaderPanelEmptyStateView.swift

```diff
@@ -0,0 +1,166 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Common
+
+class ReaderPanelEmptyStateView: UIView {
```

Comment:
The content of this class already exists in the project. I simply extracted it to a new file.